### ISSUE TITLE
[BUG]下载成功率低

### ISSUE THREAD

**Issue by github-zhanglei** [Johnserf-Seed/TikTokDownload#505]
**描述出现的错误**

**bug复现**
只有一个下载成功了，请参考下面截图

**截图**
![image](https://github.com/Johnserf-Seed/TikTokDownload/assets/47292701/500cd58a-6647-4524-abbc-5f1b8bec6bbb)

**桌面（请填写以下信息）：**
-操作系统：[例如windows11 64bit]
-项目版本：[如1.4.0]
-py版本：[如3.11.4]

**附文**
在此处添加有关此问题的其他备注。

**Comment by Johnserf-Seed** [comment]
预计下载时间结束了还是这样吗？

**Comment by daymade** [comment]
> 预计下载时间结束了还是这样吗？

是的, 我从 13000 版本升级到 14000 版本之后, 以前可以下载的作者很多都无法正常下载了, 应该是 14000 引入了异步下载优化的原因, 最近几个下载失败的 issue 都是相同的原因, 我已经提交了一个 [PR](https://github.com/Johnserf-Seed/TikTokDownload/pull/506) 改动来修复这个问题, 请作者帮忙 Review 一下, 谢谢
